### PR TITLE
R89

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

**Comment by BenFradet** [comment]
@alexanderdean finally got around to finishing the first draft in the middle of all this travis mess, input welcome! :+1:

File: _posts/2017-05-10-snowplow-r89-plain-of-jars-released.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+- [Gabor Ratky][rgabo] of Secret Sauce Partners for his great help during QA
+- [Phil Kallos][pkallos], formerly of Popsugar, for breaking ground on this port and his support throughout the coding process
+- [David White][13scoobie] of Nordstrom for his guidance and encouragement
+- everyone else who followed along on our journey
+
+This has been one of the most inclusive and collaborative Snowplow releases in our history - an exciting outcome of our [evolving RFC process][rfcs], and one on which bodes well for the future as we continue to tackle exciting new features and major refactorings. Thank you all!
+
+<h2 id="why-spark">2. Why Spark?</h2>
+
+ADD HISTORY OF SCALDING AND TWITTER
```
Comment (BenFradet) [inline _posts/2017-05-10-snowplow-r89-plain-of-jars-released.md]: What did you have in mind?

Just a brief intro to what Scalding is?

File: _posts/2017-05-10-snowplow-r89-plain-of-jars-released.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+This release ports two core batch pipeline components over from Scalding to Spark:
+
+1. Scala Hadoop Enrich, being renamed Spark Enrich
+2. Scala Hadoop Shred, being renamed Relational Database Shredder
+
+Spark Enrich effectively replaces Scala Hadoop Enrich. It is a "lift and shift" port, having the exact same set
+of functionalities and acting as a drop-in replacement.
+
+For its part, Relational Database Shredder is the successor to Scala Hadoop
+Shred. XXXXXXXX Here too, the feature set of Scala Hadoop Shred has been preserved. It'll
```
Comment (BenFradet) [inline _posts/2017-05-10-snowplow-r89-plain-of-jars-released.md]: ?

File: _posts/2017-05-10-snowplow-r89-plain-of-jars-released.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+For more details on this release, please check out the
+[release notes][snowplow-release] on GitHub.
+
+If you have any questions or run into any problems, please
+[raise an issue][issues] or get in touch with us through
+[the usual channels][talk-to-us].
+
+[plain-of-jars]: https://en.wikipedia.org/wiki/Plain_of_Jars
+[plain-of-jars-img]: /assets/img/blog/2017/02/chichen-itza-mexico.jpg
```
Comment (BenFradet) [inline _posts/2017-05-10-snowplow-r89-plain-of-jars-released.md]: I don't know how you usually pick an image

File: _posts/2017-05-10-snowplow-r89-plain-of-jars-released.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+[snowplow-release]: https://github.com/snowplow/snowplow/releases/r89-plain-of-jars
+
+[pkallos]: https://github.com/pkallos
+[13scoobie]: https://github.com/13scoobie
+[rgabo]: https://github.com/rgabo
+
+[rfcs]: xxx
```
Comment (BenFradet) [inline _posts/2017-05-10-snowplow-r89-plain-of-jars-released.md]: ?

**Comment by alexanderdean** [comment]
@BenFradet - no worries, those were notes for me not for you! I'll do some more work on the blog post over the next couple of days...

**Pull request merged**